test/lib/Launcher.ts

```diff
@@ -4,6 +4,9 @@ import path from "path";
 import getPort from "./getPort";
 import waitForConnector from "./waitForConnector";
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export type ConnectorWithMetadata = ConnectorClient & { _metadata?: Record<string, string> };
```

Comment:
shouldn't this be ConnectorClientWithMetadata?

Reply:
additionally, what's wrong with the naming? (eslint disable)

Reply:
"_" start is not allowed

Reply:
wtb just metadata?

Reply:
Yea but when we get some metadata API this would crash